src/senaite/sync/browser/views.py

```diff
@@ -499,6 +538,26 @@ def get_storage(self, domain=None):
             self.storage[domain]["credentials"] = OOBTree()
         return self.storage[domain]
 
+    def reindex_updated_objects(self):
+        """
+        Reindexes updated objects.
+        """
+        total = len(self.uids_to_reindex)
+        logger.info('Reindexing {} objects which were updated...'.format(total))
+        indexed = 0
+        for uid in self.uids_to_reindex:
+            obj = api.get_object_by_uid(uid[0], None)
+            if obj is None:
+                logger.error("Object not found: {} ".format(uid[1]))
```

Comment:
ah ok, here you need the `repr(obj)`. Please do that `repr` call in `format` and put the raw obj in the second item.

Reply:
Since we already have one map for objects, I didn't want to duplicate but kept strings. Also this is very rare to happen, so maybe no need to increase the memory by twice?

Reply:
Indeed, keeping an eye on the memory resources is a good programmers habit, especially for Plone development. Well argued Nihad.